paymentsheet/src/main/java/com/stripe/android/paymentelement/embedded/manage/ManageActivity.kt

```diff
@@ -109,7 +109,9 @@ internal class ManageActivity : AppCompatActivity() {
         val scrollState = rememberScrollState()
         BottomSheetScaffold(
             topBar = {
-                val topBarState by screen.topBarState().collectAsState()
+                val topBarState by remember(screen) {
```

Comment:
We never tested this UI code, and it's never idle. Writing UI tests on the activity will validate this doesn't regress.